src/api/JsonApi.js

```diff
@@ -305,6 +305,49 @@ const splitDataByParent = ({
   }, {})
 }
 
+/**
```

Comment:
I am not sure this is necessary. It is better to use side loading for the workflows which makes it more maintainable. I can go through it with you.